src/Groups/libraries/libraries.jl

```diff
@@ -0,0 +1,87 @@
+export
+    moved_points,
+    number_moved_points
+
+moved_points(x::PermGroupElem) = [y for y in GAP.gap_to_julia(GAP.Globals.MovedPoints(x.X))]
+moved_points(G::PermGroup) = [y for y in GAP.gap_to_julia(GAP.Globals.MovedPoints(G.X))]
+number_moved_points(x::Union{PermGroupElem,PermGroup}) = length(moved_points(x))
```

Comment:
These should be elsewhere, e.g. in a new file `gsets.jl`

Reply:
Actually I just thought about it, and I am not sure they should be in gsets.jl anymore; they rather seem to be specific to permutations and permutation groups, but not general gsets... Anyway, can be decided later